nixos/modules/services/misc/gitea.nix

```diff
@@ -477,6 +477,7 @@ in
       in ''
         # copy custom configuration and generate a random secret key if needed
         ${optionalString (cfg.useWizard == false) ''
+          umask 027
```

Comment:
You don't want to run this in a sub-shell?

Reply:
AFAIU the only file creation operations only happen within this block, so unless I'm missing something right now, this shouldn't be needed.

Reply:
That will cause problems though if another file creation has to be added at a later point.

Reply:
@dotlambda updated.